pandas/tests/extension/base/groupby.py

```diff
@@ -67,3 +67,16 @@ def test_groupby_extension_apply(self, data_for_grouping, op):
         df.groupby("B").A.apply(op)
         df.groupby("A").apply(op)
         df.groupby("A").B.apply(op)
+
+    def test_in_numeric_groupby(self, data_for_grouping):
+        df = pd.DataFrame({"A": [1, 1, 2, 2, 3, 3, 1, 4],
+                           "B": data_for_grouping,
+                           "C": [1, 1, 1, 1, 1, 1, 1, 1]})
+        result = df.groupby("A").sum().columns
```

Comment:
how does this test pass? (I thought reductions did not yet work for EAs?)

Reply:
I wonder if it's going through the fallback `np.mean(arr)`? Will investigate.

Reply:
Yeah, so reductions for Series fail (no `_reduce`), but for dataframe it seems to use some ndarray fallback